otcextensions/sdk/dws/v1/_proxy.py

```diff
@@ -341,3 +342,71 @@ def wait_for_cluster_scale_out(self, cluster, interval=5, wait=1800):
         if is_snapshotting:
             time.sleep(60)
             self.wait_for_cluster(cluster, interval, wait)
+
+    def list_cluster_tags(self, cluster):
+        """
+        List tags for a DWS cluster.
+
+        :param cluster: Cluster ID or an instance of
+            `otcextensions.sdk.dws.v1.cluster.Cluster`.
+        :returns: List of `otcextensions.sdk.dws.v1.tag.Tag` instances.
+        """
+        cluster = self._get_resource(_cluster.Cluster, cluster)
+        return self._list(_tag.Tag, cluster_id=cluster.id)
+
+    def create_cluster_tag(self, cluster, tag):
+        """
+        Create a new tag for a DWS cluster.
+
+        :param cluster: Cluster ID or an instance of
+            `otcextensions.sdk.dws.v1.cluster.Cluster`.
+        :param tag: Dictionary with 'key' and 'value' for the tag.
+        :returns: Created tag instance.
+        """
+        cluster = self._get_resource(_cluster.Cluster, cluster)
+        return self._create(_tag.Tag, cluster_id=cluster.id, **tag)
+
+    def delete_cluster_tag(self, cluster, tag_key, ignore_missing=True):
+        """
+        Delete a single tag from a DWS cluster.
+
+        :param cluster: Cluster ID or an instance of
+            `otcextensions.sdk.dws.v1.cluster.Cluster`.
+        :param tag_key: The key of the tag to be deleted.
+        :param ignore_missing: When False,
+            `openstack.exceptions.ResourceNotFound`
+            will be raised when the tag does not exist.
+            When True, no exception will be set when attempting
+            to delete a nonexistent tag.
+        :returns: None
+        """
+        cluster = self._get_resource(_cluster.Cluster, cluster)
+        tag_obj = _tag.Tag.existing(key=tag_key, cluster_id=cluster.id)
+        return self._delete(
+            _tag.Tag, tag_obj.key, cluster_id=cluster.id,
+            ignore_missing=ignore_missing
+        )
+
+    def batch_create_cluster_tags(self, cluster, tags):
+        """
+        Batch addition of tags for a cluster.
+
+        :param cluster: Cluster ID or an instance of
+            `otcextensions.sdk.dws.v1.cluster.Cluster`.
+        :param tags: List of dictionaries of tags to add.
+        :return: Response object.
+        """
+        cluster = self._get_resource(_cluster.Cluster, cluster)
+        return _tag.Tag().manage_tags_batch(self, cluster.id, tags, 'create')
+
+    def batch_delete_cluster_tags(self, cluster, tags):
+        """
+        Batch deletion of tags from a cluster.
+
+        :param cluster: Cluster ID or an instance of
+            `otcextensions.sdk.dws.v1.cluster.Cluster`.
+        :param tags: List of dictionaries of tags to delete.
+        :return: Response object.
```

Comment:
should be None

Reply:
or object Tag, not response

Reply:
Changed response to 'otcextensions.sdk.dws.v1.tag.Tag'